src/Chirp.Web/Pages/Auth.cshtml.cs

```diff
@@ -1,11 +1,9 @@
 using Microsoft.AspNetCore.Authorization;
-using Microsoft.AspNetCore.Mvc;
-using Microsoft.AspNetCore.Mvc.RazorPages;
 
 namespace Chirp.Web.Pages;
 
 [Authorize]
-public class Auth : PageModel
+public class AuthModel : PageModel
```

Comment:
Auth is not really a "model" though. We just use it as an intermediate layer to do authentication.

Reply:
Sure, I just thought to keep the naming convention consistent by naming all classes that inherit PageModel: ____Model